pkg/glance/dbsync.go

```diff
@@ -27,19 +27,20 @@ import (
 
 const (
 	// DBSyncCommand -
-	DBSyncCommand = "/usr/local/bin/kolla_set_configs && su -s /bin/sh -c \"glance-manage db sync\" glance"
+	//DBSyncCommand = "/usr/local/bin/kolla_set_configs && su -s /bin/sh -c \"glance-manage db sync\" glance"
+	DBSyncCommand = "su -s /bin/sh -c \"glance-manage db sync\" glance"
 )
 
 // DbSyncJob func
 func DbSyncJob(
 	instance *glancev1.Glance,
 	labels map[string]string,
 	annotations map[string]string,
+	confSecrets []string,
 ) *batchv1.Job {
 	runAsUser := int64(0)
 
-	dbSyncExtraMounts := []glancev1.GlanceExtraVolMounts{}
-	secretNames := []string{}
+	//dbSyncExtraMounts := []glancev1.GlanceExtraVolMounts{}
```

Comment:
nit: We can remove line 43 if its not relevant anymore.

Reply:
It's actually needed, I was just experimenting with this draft on a couple of approaches.